website/content/v1.11/introduction/what-is-new/index.md

```diff
@@ -6,4 +6,157 @@ description: "Discover the latest features and updates in Talos Linux 1.11."
 
 For critical changes, refer to the [upgrade notes]({{< relref "../talos-guides/upgrading-talos" >}}).
 
-TBD
+## Disk Management
+
+### Raw and Existing Volumes
+
+In addition to the existing support for [user volumes]({{< relref "../talos-guides/configuration/disk-management/user" >}}), Talos now supports two new types of volumes:
+
+* [raw volumes]({{< relref "../talos-guides/configuration/disk-management/raw" >}}) - allows to allocate unformatted disk space as a partition.
+* [existing volumes]({{< relref "../talos-guides/configuration/disk-management/existing" >}}) - allows to use existing data partitions or disks.
+
+### Swap and Zswap
+
+Talos now supports [swap]({{< relref "../talos-guides/configuration/swap" >}}) on block devices, and `zswap`, a compressed cache for swap pages.
+
+### Disk Encryption
+
+[Disk encryption]({{< relref "../talos-guides/configuration/disk-encryption" >}}) for system volumes is now managed by the [`VolumeConfig`]({{< relref "../reference/configuration/block/volumeconfig" >}}) machine configuration document.
+Legacy configuration in `v1alpha1` (`.machine.systemDiskEncryption`) machine configuration is still supported.
+
+New per encryption key option `lockToSTATE` is added to the `VolumeConfig` document, which allows to lock the volume encryption key to the secret salt in the `STATE` volume.
+So, if the `STATE` volume is wiped or replaced, the volume encryption key will not be usable anymore.
+
+### Disk Wipe
+
+Talos now supports `talosctl disk wipe` command in maintenance mode (`talosctl disk wipe <disk> --insecure`).
+
+## SBOMs
+
+Talos now publishes [Software Bill of Materials (SBOM)]({{< relref "../advanced/sbom" >}}) in the SPDX format.
+
+## Platform Updates
+
+### Azure
+
+Talos on Azure now defaults to MTU of 1400 bytes for the `eth0` interface to avoid packet fragmentation issues.
+The default MTU can be overridden with machine configuration.
+
+### VMware
+
+Talos VMWare platform now supports `arm64` architecture in addition to `amd64`.
+
+### Boot
```

Comment:
probably a note about the UEFI boot entry named `Talos UKI` too

Reply:
the fact that we install oursevles into the boot order, yes...